report_aeroo/report_aeroo.py

```diff
@@ -113,6 +114,12 @@ def run_subprocess(self, cr, uid, ids, report_xml, temp_file, cmd):
                         'output_format': output_format,
                     })
 
+    def _kill_process(self, proc):
```

Comment:
@dufresnedavid It would be more robust to try a sigterm first and then a sigkill. Processes that are forcefully killed tend to leave garbage behind (temp files, lock files, etc) so in the long run you may end with a filesystem full of them.